pkg/scheduler/framework/plugins/nodevolumelimits/non_csi_test.go

```diff
@@ -31,8 +31,8 @@ import (
 	utilpointer "k8s.io/utils/pointer"
 )
 
-func TestAzureDiskLimits(t *testing.T) {
-	oneVolPod := &v1.Pod{
+var (
```

Comment:
The same variables were being defined for the same values in each test, so they were defined together on top of the file.

Reply:
(This change is not directly related to this PR, so move to #103547)